src/providers/proxy/proxy_id.c

```diff
@@ -1488,6 +1488,152 @@ static int get_initgr(TALLOC_CTX *mem_ctx,
     return ret;
 }
 
+static int compare_group_members(struct proxy_id_ctx *ctx,
+                       TALLOC_CTX *tmp_ctx,
+                       struct sss_domain_info *dom,
+                       long int num_gids,
+                       gid_t *gids,
+                       char *db_groupname,
+                       bool *group_found)
```

Comment:
Out args should have `_` prefix ( https://sssd.io/contrib/coding-style.html#additional-notes )

Reply:
This is no longer necessary for `group_found` but I've taken into account the improvement for other variables. Thanks!